bots/validatespeakerscornerissue.py

```diff
@@ -0,0 +1,118 @@
+import os
+from datetime import date, timedelta
+from github import Github
+import arxiv
+
+VSF_BOT_TOKEN = os.getenv("VSF_BOT_TOKEN")
+ISSUENUMBER = os.getenv("ISSUENUMBER")
+
+g = Github(VSF_BOT_TOKEN)
+repo = g.get_repo("virtualscienceforum/virtualscienceforum")
+issue = repo.get_issue(number=ISSUENUMBER)
+
+def CheckDate(dateStr, timeStr):
+    scheduledDate = date.fromisoformat(dateStr)
+    if( (scheduledDate - date.today()).days < 14 ):
+        return False, "Please schedule your talk at least two weeks into the future"
+
+    return True, "Ok"
+
+def CheckArxiv(preprintID):
+    # 1) Check if preprint exists
+    arxivResult = arxiv.query(id_list=[preprintID])
+
+    preprint = {}
+    if( len(arxivResult) == 0 ):
+        return False, preprint
+
+    # Extract the paper
+    preprint["title"] = arxivResult[0].title.replace('\n ', '')
+    preprint["abstract"] = arxivResult[0].summary
+    preprint["authors"] = arxivResult[0].authors
+
+    return True, preprint
+
+def UpdateIssue(currentBody, preprint):
+    b = currentBody
+
+    # Extract the current title and abstract
+    title = b.split("Title")[1].split()[0]
+    abstract = b.split("Abstract")[1].split()[0]
+
+    # If the title has been left empty, update it
+    if (title == "<!--"):
+        before, after = b.split("Title")
+        b = before + "Title\n\n" + preprint["title"] + after
+
+    # If the abstract has been left empty, update it
+    if (abstract == "<!--"):
+        before, after = b.split("Abstract")
+        b = before + "Abstract\n\n" + preprint["abstract"] + after
+
+    # Add the authors
+    currentNotes = b.split("Notes")[1].split()[0]
+    currentNotes = "Authors: " + ", ".join(preprint["authors"])
+    before, after = b.split("Notes")
+    b = before + "Notes\n\n" + currentNotes + after
+
+    with open("updatesissue.txt", "w") as f:
+        f.write(b)
+
+    issue.edit(body=b)
+
+def AddComments(comments):
+    commentBody = "I am sorry, I was unable to successfully process your data.\n"
+    commentBody += "Please amend the following issues: \n"
+
+    for c in range(len(comments)-1):
+        commentBody += "* " + comments[c] + "\n"
+    commentBody += "* " + comments[-1]
+
+    issue.create_comment(commentBody)
+
+def AddTeamChecklist():
```

Comment:
Instead of this function we should create a constant `TEAM_CHECKLIST`, and directly call `issue.create_comment(TEAM_CHECKLIST)`.